### DISCUSSION TITLE
Added Documentation

### DISCUSSION THREAD

**Discussion by lkoepsel** [bloombloombloom/Bloom#21] [Show and tell]
This certainly isn't an issue...

I wanted to let you know I posted an update to using avr-gdb and Bloom on my site. I write both curriculum and for my site, they tend to blur into one. 

Its not finished as I still need to update how to program DWEN w/o a Dragon, test out the short-term fix and advise as to how to use the short-term fix to upload code. Its a start...

https://wellys.com/posts/avr_c_gdbbloom/

Do with as you wish.
Lief

**Comment by navnavnav** [comment]
Thanks for sharing this with me, Lief. It's an excellent resource and it's great to see others sharing information on Bloom.

Just FYI, the new version due to be released later this week will introduce memory inspection panes. These panes will allow users to inspect target memories via the Insight window. See my latest comment at  https://github.com/navnavnav/Bloom/issues/7#issuecomment-1001316663. There, I go into detail of the changes to the Insight window. Hopefully your students will find those new functions useful.

If you have any questions or ever wish to clarify anything, feel free to ask 👍🏽
